### PR TITLE
Combined new-page, upload, and connect dialog

### PR DESCRIPTION
**Pull request by mworrell**
### Description

In the admin, combine the "new page", "upload", and "connect" tabs in one dialog.

This also adds a list of existing resources matching the title being typed.

### Checklist

- [ ] documentation updated
- [ ] tests added
- [ ] no BC breaks

### CODE REVIEW

**Comment by mworrell** [comment]
Example screen shot when opened from the 'media' page:

Screen Shot 2019-03-11 at 15 26 37

**Comment by mworrell** [comment]
Now with more actions in the list of matching pages. The 'preview' button opens an overlay with the resource for inspection.

Screen Shot 2019-03-12 at 09 22 48

**Comment by ddeboer** [comment]
In terms of user flow, wouldn’t it be better to show the search results first and below or besides that a big ‘Create new page’ button which then shows the edit fields?

**Comment by mworrell** [comment]
I think this format with the create left, and the existing content on the right is quite ok. We show this if the user wants to create content, and with the search we want to help prevent making duplicate content.

File: modules/mod_admin/templates/_action_dialog_new_rsc_tab_find_results_item.tpl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                <p>{{ id|summary:80 }}</p>
+            {% endblock %}
+            {% block item_actions %}
+            <p class="rsc-actions">
+                <a href="#" class="btn btn-default action-preview">{_ Preview _}</a>
+                {% if predicate and is_linkable %}
+                    <a href="#" class="btn btn-primary action-connect">{_ Connect _}</a>
+                {% endif %}
+                {% if id.is_editable %}
+                    <a href="#" class="btn btn-primary action-edit">{_ Edit _}</a>
```
[the remark this answers is not in the document]
Comment (mworrell) [reply modules/mod_admin/templates/_action_dialog_new_rsc_tab_find_results_item.tpl]: Changed it.

**Comment by ddeboer** [comment]
Just to be clear: this adds a new findnew tab as an alternative to the find and new tabs, making findnew the default?

**Comment by mworrell** [comment]
Yes

**Pull request merged**